### PR TITLE
[webui] Some minor UI tweaks and new Dashboard tab created

### PR DESCRIPTION
**Pull request by saganshul**
As discussed, this PR creates a new tab in navigation bar, dashboard. This PR also do some minor UI changes like font and colors.

## Reviewers
@delta24 @7a @tunnelshade @anantshri 

## Screenshots (if appropriate):
screenshot from 2016-05-31 23-39-39
screenshot from 2016-05-31 23-39-27

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

### Checklist:

- [ ] My code follows the code style (modified PEP8) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: includes/css/stylesheet.css [hunk 1]
```diff
@@ -86,4 +97,62 @@ body {
...
+}
+
+.navbar-inverse {
+    background-color: #2f4050;
+    border-color: #2f4050;
+}
+
+.navbar-inverse .navbar-brand {
+    color: #fff;
+		font-family: 'JosefinSans', sans-serif;
```
Comment (delta24) [inline includes/css/stylesheet.css]: Keep "OpenSans" everywhere for now.

**Comment by delta24** [comment]
A suggestion to keep clicks to a minimum:
Use the split button dropdown available here http://getbootstrap.com/components/#btn-dropdowns-split
for the "Targets" link in the navbar. The button goes straight to the Targets page and dropdown shows the recently run targets.

What do you think? :)

**Comment by saganshul** [comment]
@delta24 I think its great idea. I will make it as soon as possible.

**Comment by delta24** [comment]
Otherwise good to merge. :+1:

**Pull request merged**